datasets/steam/setup_steam_instance/script.js

```diff
@@ -0,0 +1,26 @@
+const { MeiliSearch } = require('meilisearch');
+const documents = require('../steam-games.json');
+const settings = require('../settings.json');
+
+const client = new MeiliSearch({
+  host: 'localhost:7700',
+  apiKey: 'masterKey',
+});
+
+const setup_steam_instance = async () => {
+  try {
+    const uid = 'steam-video-games';
+
+    const task = await client.createIndex(uid);
+    await client.waitForTask(task.uid);
+    const index = await client.getIndex(uid);
+
+    await index.updateSettings(settings);
+
+    await index.addDocuments(documents);
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+setup_steam_instance();
```

Comment:
```suggestion
(async () => {
  try {
    const uid = 'steam-video-games';
    
    await client.index(uid).updateSettings(settings);
    await client.index(uid).addDocuments(documents);
  } catch (e) {
    console.error(e);
  }
})();
```

Reply:
Lets use an [iife](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) and reduce the code.

With the new task system we can just throw all tasks without having to wait for the index to be created! 
`updateSettings` creates the index and updates the settings.
Once the task of `updateSettings` is done it will start the one from `addDocuments` in which the index will already be created.